### PR TITLE
rawoptions for packages and classes

### PR DESCRIPTION
**Pull request by FrankMittelbach**
# Internal housekeeping

## Status of pull request

- To be finished by others - wanted  :-)
- Under development

it works across the test suite (or did a minute ago), but it is missing a paragraph in ltnews33 (so far only a placeholder there) and maybe there are other things that are needed - I only looked that it runs again.

But once ltnews33 is written it can probably be merged.

## Checklist of required changes before merge will be approved
- [n/a] Test file(s) added
- [x] Version and date string updated in changed source files
- [x] Relevant `\changes` entries in source included
- [x] Relevant `changes.txt` updated
- [x] `ltnewsX.tex` updated

### CODE REVIEW

**Comment by davidcarlisle** [comment]
@FrankMittelbach  Thanks for fixing the merge conflicts I have resynced locally and will make some tests here...

**Comment by davidcarlisle** [comment]
I added some ltnews text and two test files, looks good to merge if travis tests complete OK

File: base/doc/ltnews33.tex [hunk 1]
```diff
@@ -355,6 +355,41 @@ \subsection{Execute \cs{par} at the end of \cs{marginpar} arguments}
...
+changed however the original un-normalised class and option lists
+are now saved. they are not used in the standard processing, however
+extended package option systems may use these ``raw'' option list
+macros if they are defined.
+
+The one change affecting the standard processing is that the low level
+error mentioned above is now avoided as values (any tokens to the
+right of an =  sign) are removed from consideration from the ``unused
+option list''  In this release \texttt{clip=true}` and
+`\texttt{clip=false}` both contribute \texttt{clip} to the list of
```
Comment (muzimuzhi) [inline base/doc/ltnews33.tex:386]: A missing period just after ` ``unused option list'' `, and a missing `` ` `` just before `` \texttt{clip=true}` ``.

**Review by muzimuzhi — COMMENTED** [review]
some typos in `ltnews33.tex`

**Pull request merged**